src/systemctl/systemctl-set-environment.c

```diff
@@ -120,13 +120,19 @@ int import_environment(int argc, char *argv[], void *userdata) {
 
         if (argc < 2) {
                 _cleanup_strv_free_ char **copy = NULL;
+                char **e;
 
                 copy = strv_copy(environ);
                 if (!copy)
                         return log_oom();
 
                 strv_env_clean_with_callback(copy, invalid_callback, NULL);
 
+                STRV_FOREACH(e, copy)
+                        if (string_has_cc(*e, NULL))
+                                log_info("Environment variable $%.*s contains control characters, importing anyway.",
```

Comment:
`log_notice()` maybe? it's more than informational, but less than a warning, so notice sounds right?